frontend/controller/actions/chatroom.js

```diff
@@ -178,11 +178,11 @@ export default (sbp('sbp/selectors/register', {
     const originatingContractID = state.attributes.groupContractID ? state.attributes.groupContractID : contractID
 
     // $FlowFixMe
-    return Promise.all(Object.keys(state.members).map((pContractID) => {
-      const CEKid = findKeyIdByName(rootState[pContractID], 'cek')
+    return Promise.all(Object.keys(state.members).map(async (pContractID) => {
+      const CEKid = await sbp('chelonia/contract/currentKeyIdByName', pContractID, 'cek')
```

Comment:
Why is this calling `await`?

Reply:
Because of sandboxing in the future. All `chelonia/` calls may be async.